sci-mathematics/minisat/files/minisat-2.2.0_p20130925-nusmv.patch

```diff
@@ -0,0 +1,874 @@
+diff -Naur minisat-37dc6c67e2af26379d88ce349eb9c4c6160e8543_ORIG/external/inttypes.h minisat-37dc6c67e2af26379d88ce349eb9c4c6160e8543/external/inttypes.h
+--- minisat-37dc6c67e2af26379d88ce349eb9c4c6160e8543_ORIG/external/inttypes.h	1970-01-01 01:00:00.000000000 +0100
++++ minisat-37dc6c67e2af26379d88ce349eb9c4c6160e8543/external/inttypes.h	2015-07-10 15:37:48.719358633 +0200
```

Comment:
@g-braeunlich understandable, but I don't like installing standard headers, lest they interfere with other headers. And even Gentoo Prefix on Windows/Interix won't use MSVC, so we can safely nix it.